test/e2e/kubectl.go

```diff
@@ -597,13 +609,14 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 			Expect(err).To(ContainSubstring("GET http://invalid/api"))
 
 			By("trying to use kubectl with invalid namespace")
-			output, _ := framework.RunHostCmd(ns, simplePodName, "/kubectl get pods --namespace=invalid --v=6 2>&1")
-			Expect(output).To(ContainSubstring("No resources found"))
+			output, err := framework.RunHostCmd(ns, simplePodName, "/kubectl get pods --namespace=invalid --v=6 2>&1")
 			Expect(output).ToNot(ContainSubstring("Using in-cluster namespace"))
 			Expect(output).To(ContainSubstring("Using in-cluster configuration"))
 			if matched, _ := regexp.MatchString(fmt.Sprintf("GET http[s]?://%s:%s/api/v1/namespaces/invalid/pods", inClusterHost, inClusterPort), output); !matched {
 				framework.Failf("Unexpected kubectl exec output: ", output)
 			}
+			// we expect to get a forbidden since we don't have rights to the invalid namespace
+			Expect(err).To(ContainSubstring("Response Status: 403 Forbidden"))
```

Comment:
> Also if rbac is disabled? Is this test expected to work then?

removed. There's a test to make sure we hit the right namespace based on the arg, which is the point of the test, so I think that's good enough.